ingestion-beam/src/main/java/com/mozilla/telemetry/aet/AetProcessing.java

```diff
@@ -0,0 +1,59 @@
+package com.mozilla.telemetry.aet;
```

Comment:
This package is missing a corresponding test.

Reply:
Test added.